crypto/asn1/a_gentm.c

```diff
@@ -157,10 +157,10 @@ int ASN1_GENERALIZEDTIME_print(BIO *bp, const ASN1_GENERALIZEDTIME *tm)
     if (BIO_printf(bp, "%s %2d %02d:%02d:%02d%.*s %d%s",
                    _asn1_mon[M - 1], d, h, m, s, f_len, f, y,
```

Comment:
> I'm directly opposed to having to separate source copies of the same array.

Complexity is obviously a parameter. You surely wouldn't oppose to idea of having `const int i = 42;` in multiple files. Is this actually very much different? I do subscribe to idea of maintenance costs, but what are the chances of screwing this one up? Or getting extra month? Or having one renamed? And on the other hand there is something you actually *gain* [by having multiple copies]. You can read the piece of code without having to cross-reference it with other files.